code/ui/generalhud/menu/Menu.cs

```diff
@@ -1,21 +1,68 @@
-namespace TTTReborn.UI
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+namespace TTTReborn.UI.Menu
 {
     public partial class Menu : RichPanel
     {
         public static Menu Instance;
 
-        private DraggableHeader _draggableHeader;
+        private MenuHeader _menuHeader;
+
+        public readonly PanelContent MenuContent;
+
+        private readonly string _baseTitle = "Main Menu";
```

Comment:
I would personally just make this an empty string by default.

Reply:
So that we don't have a title?

Reply:
Yeah, if no header is provided, no title is given. That seems a bit more intuitive than just to make it "Main menu" imo.